### PR TITLE
Ukrainian locale

### PR DESCRIPTION
**Pull request by yuliatokaryk**

### CODE REVIEW

File: app/views/authors/_form.html.erb [hunk 1]
```diff
@@ -11,27 +11,27 @@
 <div class="w-full max-w-xs">
   <%= form_with(model: @author, local:true) do |f| %>
     <div class="mb-4">
-      <%= f.label :first_name, class: "block text-gray-700 text-sm font-bold mb-2" %><br>
+      <%= f.label t('authors.fname'), class: "block text-gray-700 text-sm font-bold mb-2" %><br>
```
Comment (ihortok) [inline app/views/authors/_form.html.erb]: read this:
https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models

add translations for attributes for Author model

File: app/views/authors/index.html.erb [hunk 2]
```diff
@@ -8,25 +8,25 @@
           <thead class="border-b">
             <tr>
               <th scope="col" class="text-sm font-medium text-gray-900 px-6 py-4 border-r">
-                First name
+                <%= t('authors.fname') %>
```
Comment (ihortok) [inline app/views/authors/index.html.erb]: the same, use translations for attributes

File: app/views/authors/index.html.erb [hunk 3]
```diff
@@ -72,5 +72,5 @@
 </div>
 
 <% if user_signed_in? %>
-  <%= link_to "Add new author", new_author_path, class: "bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded" %>
+  <%= link_to t('authors.add_new'), new_author_path, class: "bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded" %>
```
Comment (ihortok) [inline app/views/authors/index.html.erb]: use this path:

```
en:
  authors:
    index:
      add_new: Add new author
```

then in the index view you can fetch the locale `t('.add_new')`

File: app/views/authors/_form.html.erb (hunk 1 above)
Comment (ihortok) [reply app/views/authors/_form.html.erb]:
```
en:
  activerecord:
    attributes:
      author:
        first_name: First name
```

**Review by ihortok — CHANGES_REQUESTED** [review]

File: app/views/authors/_form.html.erb (hunk 1 above)
Comment (yuliatokaryk) [reply app/views/authors/_form.html.erb]: https://github.com/yuliatokaryk/quotes/pull/24/commits/dc2b4fe4ba91acdf6a957323b45be257ea0fe351

File: app/views/authors/index.html.erb (hunk 2 above)
Comment (yuliatokaryk) [reply app/views/authors/index.html.erb]: https://github.com/yuliatokaryk/quotes/pull/24/commits/dc2b4fe4ba91acdf6a957323b45be257ea0fe351

File: app/views/authors/index.html.erb (hunk 3 above)
Comment (yuliatokaryk) [reply app/views/authors/index.html.erb]: https://github.com/yuliatokaryk/quotes/pull/24/commits/dc2b4fe4ba91acdf6a957323b45be257ea0fe351

**Review by ihortok — APPROVED** [review]

**Pull request merged**